client/src/main/java/com/paypal/selion/platform/grid/SeLionAppiumIOSDriver.java

```diff
@@ -76,64 +64,71 @@ public WebElement findElementBy(By by) {
     @Override
     public void doubleTap(WebElement webElement) {
         logger.entering(webElement);
-        String elementId = ((RemoteWebElement) webElement).getId();
-        Map<String, String> optionsMap = createOptionsForDoubleTap(elementId, ONE_FINGER);
-        javaScriptExecutor.executeScript(TAP_SCRIPT, optionsMap);
+        new TouchAction(this)
+                .tap(webElement).release()
+                .waitAction(100)
+                .tap(webElement).release()
+                .perform();
         logger.exiting();
     }
 
     @Override
     public void scrollToVisible(WebElement webElement) {
         logger.entering(webElement);
-        String elementId = ((RemoteWebElement) webElement).getId();
-        Map<String, String> arguments = new HashMap<String, String>();
-        arguments.put(ELEMENT, elementId);
-        javaScriptExecutor.executeScript(SCROLLTO_SCRIPT, arguments);
+        Dimension dimension = manage().window().getSize();
+        int height = (int) (dimension.getHeight() * 0.7);
```

Comment:
sorry for all those magic numbers. i tried to clean then a bit and use meaningful const instead.
it this case we need some space from around the edge, to not activate another functionality of IOS instead of scrolling. :)